corteca config get panics

```
$ corteca config get app
panic: reflect: call of reflect.Value.IsNil on struct Value

goroutine 1 [running]:
reflect.Value.IsNil(...)
        /usr/lib/go-1.26/src/reflect/value.go:1648
corteca/internal/configuration.ReadField({0xb0e7c0, 0x526bfa8ec40}, {0x7ffeb657e253?, 0xb543d4?})
        /home/kapernar/Github_kapcom01/corteca-cli/internal/configuration/configuration.go:458 +0x6e8
corteca/cmd.doGetConfigValue({0x7ffeb657e253, 0x3})
        /home/kapernar/Github_kapcom01/corteca-cli/cmd/config.go:101 +0x9c
corteca/cmd.init.func6(0x526bf9b9a00?, {0x526bf99c7b0?, 0x4?, 0xb4eb6e?})
        /home/kapernar/Github_kapcom01/corteca-cli/cmd/config.go:47 +0x31
github.com/spf13/cobra.(*Command).execute(0x114b120, {0x526bf99c780, 0x1, 0x1})
        /root/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x89b
github.com/spf13/cobra.(*Command).ExecuteC(0x114bca0)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x398
github.com/spf13/cobra.(*Command).Execute(...)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
corteca/cmd.Execute(...)
        /home/kapernar/Github_kapcom01/corteca-cli/cmd/root.go:77
main.main()
        /home/kapernar/Github_kapcom01/corteca-cli/main.go:10 +0x1b
```

--- a/internal/configuration/configuration.go
+++ b/internal/configuration/configuration.go
@@ -454,13 +454,13 @@ func ReadField(conf any, fieldPath string) (any, error) {
 			// edge case: first elem is empty
 			continue
 		}
-		// if field is nil
-		if field.IsNil() {
-			return nil, fmt.Errorf("cannot address nil value with key '%s'", key)
-		}
-		// if field is a pointer, dereference
+		// if field is a pointer and not nil, dereference
 		if field.Kind() == reflect.Ptr {
-			field = field.Elem()
+			if field.IsNil() {
+				return nil, fmt.Errorf("cannot address nil value with key '%s'", key)
+			} else {
+				field = field.Elem()
+			}
 		}
 		t := field.Kind()
 		switch t {

